docs/en/latest/discovery/zookeeper.md

```diff
@@ -0,0 +1,144 @@
+---
+title: nacos
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Service Discovery Via Zookeeper
+
+`Zookeeper` service discovery needs to rely on the [apisix-seed](https://github.com/api7/apisix-seed) project.
+
+### How `apisix-seed` Works
+
+![APISIX-SEED](../../../assets/images/apisix-seed.png)
+
+`apisix-seed` completes data exchange by watch the changes of `etcd` and `zookeeper` at the same time.
+
+The process is as follows:
+
+1. `APISIX` registers a route and specifies the service discovery type as `zookeeper` to `etcd`.
+2. `apisix-seed` watch the resource changes of `APISIX` in `etcd` and filters the discovery type and obtains the service name.
```

Comment:
```suggestion
2. `apisix-seed` watches the resource changes of `APISIX` in `etcd` and filters the discovery type and obtains the service name.
```

Reply:
Hi @shuaijinchao, to avoid typos, we could use https://www.grammarly.com/ to check our content :)

Reply:
thx~

Reply:
updated